cmd/scheduler/app/options/options.go

```diff
@@ -33,6 +33,11 @@ const (
 
 	defaultQPS   = 50.0
 	defaultBurst = 100
+
+    // Default arameters to control the number of feasible nodes to find and score
```

Comment:
s/arameters/parameters ?

Reply:
Fixed the errors. make verify and unit-test passed. Thanks.